crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,8 +285,11 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
-          file.to_s =~ /knife\/databags\/crowbar/
+        return false if file.basename.to_s =~ /^template-(.*).json$/ || \
+          !file.to_s =~ /knife\/databags\/crowbar/
```

Comment:
Style/MultilineOperationIndentation: Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.